01_comet/lib/features/insights/insights_model.dart

```diff
@@ -27,15 +27,14 @@ $description
   }
 }
 
-class InsightsOutput {
-  static String formatInsights(List<Insights> insights) {
-    if (insights.isEmpty) {
+extension InsightsExtension on List<Insights> {
```

Comment:
This could be even `extension type` as `Insights` type.

Reply:
Well Done @leehack  !